hw05-test-framework/src/main/java/ru/otus/hw/Main.java

```diff
@@ -0,0 +1,13 @@
+package ru.otus.hw;
+
+import ru.otus.hw.custom.framework.CustomFramework;
+import ru.otus.hw.custom.framework.CustomFrameworkException;
+
+import java.lang.reflect.InvocationTargetException;
+
+public class Main {
+
+    public static void main(String... args) throws ClassNotFoundException, NoSuchMethodException, InvocationTargetException, InstantiationException, IllegalAccessException, CustomFrameworkException {
```

Comment:
Очень много исключений. 
Лучше поймать их все внутри `CustomFramework` и обернуть в свое исключение.
```suggestion
    public static void main(String... args) throws CustomFrameworkException {
```